addons/hr/wizard/mail_activity_schedule.py

```diff
@@ -56,3 +78,12 @@ def _compute_department_id(self):
                 wizard.department_id = False if len(all_departments) > 1 else all_departments
             else:
                 wizard.department_id = False
+
+    def _schedule_activity(self, record, template, responsible, date_deadline, summary=None):
+        if self.res_model == 'hr.employee' and template.plan_id:
+            record.activity_plans_ids |= self.plan_id
+            summary = f"{template.plan_id.name} - {template.summary if template.summary else template.activity_type_id.name}"
+            activity = super()._schedule_activity(record, template, responsible, date_deadline, summary=summary)
+            activity.plan_id = self.plan_id
```

Comment:
Should be done directly in mail application. This override seems strange. If it's just about the activity summary to be overridden, I would rather define a method just for that.